pkg/resourceinterpreter/defaultinterpreter/revisereplica_test.go

```diff
@@ -71,7 +71,7 @@ func TestReviseDeploymentReplica(t *testing.T) {
 			if err == nil && tt.expectError == true {
 				t.Fatal("expect an error but got none")
 			}
-			if err != nil && tt.expectError != true {
+			if err != nil && tt.expectError == false {
```

Comment:
I still can't figure out the benefits of this change.
There is no difference between the two styles.

Reply:
@RainbowMango Ok, thanks. I'll close this pr